osu.Game/Screens/Play/SkipOverlay.cs

```diff
@@ -177,6 +165,19 @@ protected override void Update()
             isClickable = progress > 0;
             button.Enabled.Value = isClickable;
             buttonContainer.State.Value = isClickable ? Visibility.Visible : Visibility.Hidden;
+
+            if (isClickable && skipQueued && gameplayClock.CurrentTime > nextAutoClickTime)
+            {
+                if (button.Enabled.Value)
+                {
+                    button.TriggerClick();
+                    nextAutoClickTime = gameplayClock.CurrentTime + auto_click_timer;
+                }
+                else
+                {
+                    skipQueued = false;
+                }
+            }
```

Comment:
Is this change related to the fix at all? Was there anything wrong with the old implementation?

Judging by https://github.com/ppy/osu/pull/23627/commits/464b3bfdc319b7cfd7215317231232cd8fd58ea5 it seems to me that you wrongly removed the old implementation thinking it's not necessary, but then rewrote it in your own another way. If that is correct then I'd rather you put the old version back to not muddle this diff (and not reinvent what `Scheduler.AddDelayed()` is supposed to be doing anyways).